src/main/scala/no/ndla/draftapi/service/ConverterService.scala

```diff
@@ -404,7 +404,7 @@ trait ConverterService {
                                            article.tags,
                                            article.introduction,
                                            article.metaDescription,
-                                           article.metaImage,
+                                           article.metaImage.getOrElse(None),
```

Comment:
Litt usikker på denne, er det greit måte å gjøre det på eller burde det gjøres annerledes?

Reply:
Det er en grei måte å gjøre det på OM vi vil at dette skal være funksjonaliteten.
Jeg syns kanskje det blir mer riktig å håndtere `null` som om feltet er definert though.

Jeg tenker den kan skrives om til noe sånt for å få med `metaImage: null` som at vi prøver å modifisere et language felt:
```
    private def languageFieldIsDefined(article: api.UpdatedArticle): Boolean = {
      val metaImageExists = article.metaImage.map(_.isDefined).getOrElse(true)
      val langFields: Seq[Option[_]] = Seq(
        article.title,
        article.content,
        article.tags,
        article.introduction,
        article.metaDescription,
        article.visualElement
      )
      langFields.foldRight(false)((curr, res) => res || curr.isDefined) || metaImageExists
    }
```

Eller kanskje noe så dirty som det her om du har lyst til å være en dirty boy:
```
    private def languageFieldIsDefined(article: api.UpdatedArticle): Boolean = {
      val langFields: Seq[Option[_]] = Seq(
        article.title,
        article.content,
        article.tags,
        article.introduction,
        article.metaImage.getOrElse(Some(true)),
        article.metaDescription,
        article.visualElement
      )
      langFields.foldRight(false)((curr, res) => res || curr.isDefined)
    }
```

Reply:
Valgte den første, den andre er litt 2dirty5me